cypress/e2e/internal/returns-requirements/no-returns-required.cy.js

```diff
@@ -0,0 +1,160 @@
+
+    cy.get('#password').type(Cypress.env('defaultPassword'))
+
+    // click Sign in Button
+    cy.get('form > .govuk-button').click()
+
+    // assert the user signed in and we're on the search page
+    cy.contains('Search')
+
+    // search for a licence
+    cy.get('#query').type('AT/TEST/01')
+    cy.get('.search__button').click()
+    cy.get('.govuk-table__row > :nth-child(1) > a').click()
+
+    // confirm we are on the licence page and select charge information tab
+    cy.contains('AT/TEST/01')
+    cy.get('#tab_charge').click()
+
+    // confirm we are on the charge information tab
+    cy.get('#charge > :nth-child(1)').contains('Charge information')
+
+    // click set up no returns requirement
+    cy.contains('No returns required').click()
+
+    // confirm we are on the start date page
+    cy.get('.govuk-fieldset__heading').contains('Select the start date for the requirements for returns')
+
+    // choose the licence version start date and click continue
+    cy.get('#licence-start-date').check()
+    cy.contains('Continue').click()
+
+    // confirm we are on the why no returns required page
+    cy.get('.govuk-fieldset__heading').contains('Why are no returns required?')
+
+    // choose returns exception and click continue
+    cy.get('#reason-2').check()
+    cy.contains('Continue').click()
+
+    // confirm we are on the check page
+    cy.get('.govuk-heading-xl').contains('Check the return requirements')
+
+    // confirm we see text about no returns required
+    cy.contains('Returns are not required for this licence')
+
+    // confirm we are seeing the details we selected
+    cy.get('[data-test="start-date"]').should('contain.text', '12 June 2023')
+    cy.get('[data-test="reason"]').should('contain.text', 'Returns exception')
+
+    // click the change link for the start date
+    cy.get('[data-test="change-start-date"]').click()
+
+    // change start date and continue
+    cy.get('#another-start-date').check()
+    cy.get('#other-start-date-day').type('02')
+    cy.get('#other-start-date-month').type('08')
+    cy.get('#other-start-date-year').type('2023')
+    cy.contains('Continue').click()
+
+    // confirm we are back on check page and see the start date changes
+    cy.get('.govuk-heading-xl').contains('Check the return requirements for Mr J J Testerson')
+    cy.get('[data-test="start-date"]').contains('2 August 2023')
+
+    // click the change link for the reason
+    cy.get('[data-test="change-reason"]').click()
+
+    // confirm we are on the why no returns required page
+    cy.get('.govuk-fieldset__heading').contains('Why are no returns required?')
+
+    // choose returns exception and click continue
+    cy.get('#reason-3').check()
+    cy.contains('Continue').click()
+
+    // confirm we are back on check page and see the reason changes
+    cy.get('.govuk-heading-xl').contains('Check the return requirements for Mr J J Testerson')
+    cy.get('[data-test="reason"]').contains('Transfer licence')
```

Comment:
We covered changing the start date and reason on the returns-required journey. I'm happy to be overruled on the reason; it is going to a different page. But changing the start date here is just a duplication of something we are already testing in another test so should be dropped.

Reply:
Removing the start date change tests but I will leave the reason page. Seems the best route